concepts/booleans/about.md

```diff
@@ -0,0 +1,77 @@
+# About
+
+A Boolean expression is a question asked of one or more values which can be
+answered with `true` or `false`. A true expression returns `1`; a false
+expression returns `0`. Also, numbers have true/false values. Zero is false`;
+non-zero is `true`. 
+
+Old C code will either use the `int` type directly for Boolean values, or the
+programmer may have defined constants such as `TRUE` and `FALSE` to map `1` and
+`0` to a Boolean representation.
+
+With version C99 the `_Bool` type was added. The `stdbool.h` header file defines
+three new Boolean symbols: `bool`; `true` and `false`. `true` maps to `1`;
+`false` maps to `0`; and `bool` maps to `_Bool`. So now, instead of directly using 
+`int`s or rolling their own Boolean mappings, programmers can include `stdbool.h`
+and automatically get Boolean symbols in their code.
+
+## Comparison Operators
+
+C supports six comparison operators: `==` (Equal), `!=` (Not equal), `>` (Greater
+than), `<` (Less than), `>=`(Greater than or equal to), `<=` (Less than or equal
+to). These operators can compare values to evaluate the relationship of values
+into a boolean value.
```

Comment:
"These operators compare the relationships between values and resolve them into boolean values."?

Reply:
I guess the bit I feel is missing in these sentences is that it is the result of the comparison operation, rather than some kind of conversion of the operands.
Maybe I am being to pedantic ?
What do you think @ryanplusplus ?

Reply:
Hey @bobahop , I think we should go ahead and get this merged. This is a minor point only so let's skip it for now.